apps/v1/zz_generated.openshift_compatibility.go

```diff
@@ -0,0 +1,77 @@
+// +build !ignore_autogenerated
+
+// Code generated by openshift-compatibility-gen. DO NOT EDIT.
+
+package v1
+
+// CompatibilityLevel is an autogenerated function, returning the OpenShift API compatibility level.
+// It is controlled by the "openshift:compatibility-gen:level" tag in types.go.
+func (in *DeploymentConfig) CompatibilityLevel() (level int) {
+	return 1
+}
+
+// Exposed is an autogenerated function, returning the true if the type is exposed as an OpenShift API.
+// It is controlled by the "openshift:compatibility-gen:exposed" tag in types.go.
+func (in *DeploymentConfig) Exposed() bool {
+	return true
+}
+
+// CompatibilityLevel is an autogenerated function, returning the OpenShift API compatibility level.
+// It is controlled by the "openshift:compatibility-gen:level" tag in types.go.
+func (in *DeploymentConfigList) CompatibilityLevel() (level int) {
+	return 1
+}
+
+// Exposed is an autogenerated function, returning the true if the type is exposed as an OpenShift API.
+// It is controlled by the "openshift:compatibility-gen:exposed" tag in types.go.
+func (in *DeploymentConfigList) Exposed() bool {
+	return true
+}
+
+// CompatibilityLevel is an autogenerated function, returning the OpenShift API compatibility level.
+// It is controlled by the "openshift:compatibility-gen:level" tag in types.go.
+func (in *DeploymentConfigRollback) CompatibilityLevel() (level int) {
```

Comment:
what's the plan with these methods?

Reply:
e2e tests, possibly alerts in the future.